Sources/StreamChatUI/CommonViews/BaseViews.swift

```diff
@@ -99,9 +99,9 @@ open class _View: UIView, Customizable {
     }
     
     open func setUp() { /* default empty implementation */ }
-    open func setUpAppearance() { /* default empty implementation */ }
-    open func setUpLayout() { /* default empty implementation */ }
-    open func updateContent() { /* default empty implementation */ }
+    open func setUpAppearance() { setNeedsLayout() }
```

Comment:
We are not calling `super.setUpAppearance/setUpLayout/updateContent` most of the time, we should fix those 🤔

Reply:
yeah, we should check all the UI components to make sure the `super` is invoked 👍 created a ticket for that https://stream-io.atlassian.net/browse/CIS-964